recipes/cox/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "cox" %}
+{% set version = "0.1.post3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: baa6efce249e4f696de66df5939066f826cebe4b97379ebab8070cc15c9f812e
+
+build:
+  number: 0
```

Comment:
Is there a reason why this recipe is not `noarch: python` ? If some `run` dependencies are missing on some platforms, that is ok - the user will need to manually instal them.